src/shop.cpp

```diff
@@ -151,7 +151,7 @@ std::string Shop::ShopEditUrl(int idx) {
 }
 
 void Shop::SubmitSingleShop() {
-    CurrentStatusUpdate status = {};
+    CurrentStatusUpdate status = {ProgramState::ShopSubmitting, 0, 0};
```

Comment:
does `CurrentStatusUpdate status = CurrentStatusUpdate();` get rid of the warning?